### ISSUE TITLE
Issues Moderation

### ISSUE THREAD

**Issue by gewure** [dogecoin/dogecoin#1962]
The recent hype has brought a sheer unbearable amount of **total offtopic Issues** into this repository, which should be moderated/closed in time.

I am reading/commenting on Issues since about 3 years in this repository and _i would like to volunteer as a moderator,_ closing clearly offtopics such as "help me i got scammed lost my doges"- Issues, so that developers times are not wasted. 

cheers,
Gewure

**Comment by danielmbradley** [comment]
I was going to create an issue for this as well, but also suggest we need some way of clearly identifying that this isn't the place to be posting these things because a lot of the posts appear to be genuinely confused as to where they can get help or info.

**Comment by xjuanc** [comment]
Agree, there has to be some moderation on this, If there is more help needed, I'm up to it.

**Comment by 0xmatt1** [comment]
+1

**Comment by caseyPav** [comment]
Also volunteer

**Comment by roman-rr** [comment]
I created [templates](https://github.com/dogecoin/dogecoin/pull/1950#issuecomment-842441402) which should stop from stupid issues creation. Await for merge.

**Comment by haxpor** [comment]
Good idea to push forward for it. But **not** just closing the issue. We should categorize the labels then mark each issue as such. Some issues are more like discussion, some are technical, some are un-related. Take [Godot project](https://github.com/godotengine/godot/issues) (see all those nice labels marked, and some convo in each issue) as a great example in which they have a dedicated person doing what I've described and pave the way for discussion to keep into topic.

**Comment by CosmoRied** [comment]
Seriously think that this is a good idea. We can't have people spamming the issues page with feature requests and ideas if there is no intention of contributing to the project. Yes it'd be nice to have x and y all up in the codebase, but you should go over to reddit and post there. Can someone just make @caseyPav  the issues moderator and be done with it.

**Comment by maheshmnj** [comment]
Agreed we basically need people on the triage team to filter out invalid issues and also have appropriate labels attached

**Comment by Danny-dK** [comment]
In an attempt to reduce the number of issues not technical or code based that the devs can actually look into, would it be helpful to adjust the template message slightly when reporting an issue? Right now when one chooses to add a bug or feature request issue it indicates:

```
For general questions about Dogecoin or wallet recovery please use one of the various communities:
* [Dogeducation on reddit](https://www.reddit.com/r/dogeducation/)
* [Discord](https://discord.com/invite/dogecoin)
```

Could this be adjusted to something like:

```
Please note that although we appreciate much wow enthusiasm for this project and welcome any much wow ideas for the future, 
GitHub is intended for code based issues / bugs / feature request that one might have with the project. If you cannot provide 
technical details or a code based example for developers to look into and possibly incorporate into the existing code, this GitHub 
is not the right place to provide your idea. For general non-code and non-technical based ideas, please use one of the various 
communities:
* [Dogeducation on reddit](https://www.reddit.com/r/dogeducation/)
* [Discord](https://discord.com/invite/dogecoin) 
We track these communities and are notified thusly of any interesting viable ideas for the future of Dogecoin. If you post non-
technical non-code based issues here, you are hampering the development as we need to sift through all issues.
```


I thought that might help (but maybe I'm too hopeful or naive).

This issue is closely related to #2110 #2191

**Comment by CosmoRied** [comment]
Pretty good idea. No matter what, open source rules should be respected & followed, even if the goal of the project began as a joke. I hate trolls too, and I'm pretty sure a bunch are swarming around here looking for some action, so harder to complete templates could be a reasonable solution.

**Comment by Danny-dK** [comment]
@CosmoRied True, although I'm sure that a lot of people just don't know the customs of GitHub or what GitHub is actually used for (or even that crypto currency tech involves massive coding). Many have jumped aboard in the past months and I'm convinced that many are just enthusiastic but unknowing of where to post their ideas, what GitHub is, or whether they even have viable sensible ideas. I can also definitely imagine that the devs are frustrated.

**Comment by haxpor** [comment]
To be fair, we cannot blame or put much pressure to incoming new people who chime in here on Github to propose ideas. It's due to this [tweet](https://twitter.com/elonmusk/status/1396916392629137409?s=20) mostly that expresses Github is also the right place to do such thing. If core dev team really communicates with Elon, just my opinion that it's better idea to clearly communicate this through him for which area Github is suitable at for contributions. Is it worth another tweet to clarify this matter for Elon? I cannot say. But yeah, better solve the problem at the root.

**Comment by Danny-dK** [comment]
@haxpor True that the tweet was not fully informative (and maybe a bit misguided). Doing another tweet would probably just increase attention again to this github and increase the issue problem at hand again. I think now it would be best to maybe have an issue moderator as suggested in this thread and inform people where possible on this github on the best practices (for example as maybe adjusting the template messages). I can understand it is frustrating to devs, but at the same time, as you say, we cannot put too much pressure / blame on the people new to GitHub and unknowing of its practices (and hopefully only a minority is trolling). 

If I could, i would contribute with actual content to dogecoin, but alas... I'm dumb.

**Comment by ReverseControl** [comment]
Thank you everyone for your concern and your support. I am now doing moderation and triage on the project issues. I will be addressing things as I have time. Please, if you see issues which need to be closed, tag me and comment on them with the reason it should be closed. I will get the notification and I will take a look at it. That would help me address the 100+ issues faster.

**Comment by gewure** [comment]
@ReverseControl may i ask how did you attain rights to moderate?
i am, as stated, active in this repo since well over 3 years - and i would really like to also moderate here!

**Comment by michilumin** [comment]
Hey guys. Am aware of this issue, and, absolutely taking note of folks who say they're willing to moderate in here.  I'd want to discuss with some of the people who are willing; we definitely would need assistance as far as bandwidth goes on this.  

Likely this will involve giving more people the Triage role, but just want to make sure we have a constant and open chain of contact with those folks and visibility on it.  @ReverseControl is kind of the avant garde for this - but @gewure @xjuanc and @caseyPav i've absolutely noted your interest in assisting here. 

@Danny-dK I do like your revision - and, we can do it - but i worry that a lot of these folks posting are doing the "TL;DR" thing and just seeing this as a help desk and won't regard it. 

@roman-rr I'll discuss templates with folks, I just worry that may constrain issue creation a bit, and again there could just be the issue of folks ignoring it anyways. 

For now, ping @ReverseControl -- i'll hopefully be reaching out to other volunteers here to see if we can get a decent moderation team going. It's an unfortunate need but a realistic one.
